tests/manual_tests/protos/DerivedApperanceProto.prot

```diff
@@ -0,0 +1,12 @@
+#VRML_SIM R2024a utf8
```

Comment:
Ahh, good catch! Fixed!

I incorrectly assumed the test suite would fail on unexpected errors. I've added an additional test case to test the correct behavior.